scripts/superdesk/datetime/datetime.js

```diff
@@ -73,6 +73,42 @@
         }
     }
 
+    DateTimeHelperService.$inject = ['moment', 'config'];
+    function DateTimeHelperService(moment, config) {
+
+        /*
+        * @param timestring 2016-03-01T04:45:00+0000
+        * @param timezone Europe/London
+        */
+        this.splitDateTime = function(timestring, timezone) {
+            var momentTS = moment.tz(timestring, timezone);
+
+            return {
+                'date': momentTS.format(config.model.dateformat),
```

Comment:
@petrjasek from [Line 190](https://github.com/superdesk/superdesk-client-core/blob/master/scripts/superdesk-authoring/metadata/metadata.js#L190) the `timestring` passed is `2016-04-20T09:45:00+0000` i.e. it contains utcoffset. This causes moment to calculate the time part wrong.